source/_patterns/10-atoms/content/image.twig

```diff
@@ -1,20 +1,24 @@
 {% set alt = alt is defined ? alt : image.caption.title %}
 {% set src = src is defined ? src : image.sources.s8.src %}
+{% spaceless %}
 <div class="image-figure__image-wrapper imageType-{{ imageType|default('article_body_fullwidth') }}">
     <img class="image-figure__image"
          alt="{{ alt }}"
          title="{{ alt }}"
          sizes="{{ sizes }}"
             {% if image.sources is defined %}
-                srcset="{% for source in image.sources %}
-                    {{ source.src }} {{ source.width }}w
-                    {{ loop.last ? '' : ',' }}
-                 {% endfor %}"
+                {% set imageSources = [] %}
+                {% for source in image.sources %}
+                    {% set imageSource = [source.src, ' ', source.width, 'w']|join() %}
```

Comment:
Parentheses can be omitted https://twig.sensiolabs.org/doc/2.x/filters/join.html
I prefer `{% set imageSource = source.src ~ " " ~ source.width ~ "w" %}` (had to look up default character for join in twig ☹️ )